### PR TITLE
Added Titles To Pages For GA

### PR DESCRIPTION
**Pull request by abdullai-t**
This pull request introduces  page titles to all pages for Google Analytics tracking.

Currently, our website lacks consistent page titles, making it challenging to accurately track user interactions and behaviour using Google Analytics. This enhancement aims to address this issue by ensuring that each page has a meaningful and descriptive title.

## DEMO

https://github.com/massenergize/frontend-portal/assets/42780120/ba696f3c-f188-41a1-80b5-f19f0eb9d985

### CODE REVIEW

File: src/components/Pages/ActionsPage/ActionsPage.js [hunk 1]
```diff
@@ -268,9 +270,14 @@ class ActionsPage extends React.Component {
...
       },
     ];
-
     return (
       <div id="test-actions-main-wrapper">
+        {Seo({
+          title: "Actions",
+          description: "",
+          url: `${window.location.pathname}`,
+          subdomain: community?.community?.subdomain,
```
Comment (BradHN1) [inline src/components/Pages/ActionsPage/ActionsPage.js:279]: this doesn't look right

Comment (abdullai-t) [reply src/components/Pages/ActionsPage/ActionsPage.js:279]: the whole thing or only that block of code ?

File: src/components/Pages/Auth/AuthEntry.js [hunk 2]
```diff
@@ -164,6 +166,12 @@ function AuthEntry({
 
   return (
     <>
+      {Seo({
+        title: 'Auth',
```
Comment (BradHN1) [inline src/components/Pages/Auth/AuthEntry.js:170]: Title "Sign-in Page" or "Sign-in"

File: src/components/Pages/Auth/Login/LoginAuth.js [hunk 3]
```diff
@@ -117,6 +119,12 @@ export default function LoginAuth(props) {
 
   return (
     <div>
+      {Seo({
+        title: 'Auth',
```
Comment (BradHN1) [inline src/components/Pages/Auth/Login/LoginAuth.js:123]: title "Sign-in"

File: src/components/Pages/Auth/Registration/SignUpAuth.js [hunk 4]
```diff
@@ -85,21 +86,35 @@ export default function SignUpAuth({
...
-        cancelRegistration={yesDeleteMyAccount}
-        createMyAccountNow={finaliseFormAndRegister}
-        loading={loading}
-        policies={policies}
-        community={community}
-        onUsernameChange={onUsernameChange}
-      />
+      <>
+        {Seo({
+          title: "Register",
```
Comment (BradHN1) [inline src/components/Pages/Auth/Registration/SignUpAuth.js:91]: Title "Sign-up"

File: src/components/Pages/AboutUsPage/AboutUsPage.js [hunk 5]
```diff
@@ -62,8 +63,16 @@ class AboutUsPage extends React.Component {
     const donateMessage =
       donatePageData && donatePageData.title ? donatePageData.title : null;
 
+      const {community} = this.props;
+
     return (
       <div className="boxed_wrapper">
+        {Seo({
+          title: "About Us",
```
Comment (BradHN1) [inline src/components/Pages/AboutUsPage/AboutUsPage.js:71]: Perhaps all titles could be: "{CommunityName} - About Us"

Comment (abdullai-t) [reply src/components/Pages/AboutUsPage/AboutUsPage.js:71]: the title will appear in the browser and GA as `subdomain | title`

**Review by BradHN1 — APPROVED** [review]
LGTM

**Pull request merged**